DOCS-2079-3.9-notification-mule-vault-kt (#1053)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/mule-credentials-vault.adoc

Though the scenarios just mentioned satisfy most of the use cases, there are situations where you need to hide the secret encryption key value from other users of the operating system, that is, you must not show the secret encryption key as a JVM parameter, since it is visible to anyone having access to the process.

> One solution for this is to implement a custom secure property placeholder that can read the encryption key in a different way, for example from a file located in a protected folder. This custom implementation must extend `org.mule.modules.security.placeholder.SecurePropertyPlaceholderModule`.

Reviewer: ```suggestion
One way to secure the secret encryption key value is to implement a custom secure property placeholder that can read the encryption key in a different way: for example, from a file located in a protected folder. This custom implementation must extend `org.mule.modules.security.placeholder.SecurePropertyPlaceholderModule`.
```

> Another solution is to set the `mule.runtime.verbose` flag to `false` at Mule startup in order to a void that encryption keys are logged with the system properties.

Reviewer: ```suggestion
Another solution is to set the `mule.runtime.verbose` flag to `false` at Mule startup, so that encryption keys are not logged with the system properties.
```
Please verify that this rewording is still technically accurate.

* Access the xref:anypoint-enterprise-security-example-application.adoc[example application] which demonstrate Anypoint Enterprise Security in action.

> * To configure password encryption in Maven see Apache's https://maven.apache.org/guides/mini/guide-encryption.html[mini-guide] on server password encryption

Reviewer: ```suggestion
* To configure password encryption in Maven, see Apache's https://maven.apache.org/guides/mini/guide-encryption.html[guide] on server password encryption.
```
